src/sentry/snuba/metrics/query.py

```diff
@@ -125,17 +126,31 @@ def validate_where(self) -> None:
     def validate_orderby(self) -> None:
         if not self.orderby:
             return
-        if len(self.orderby) > 1:
-            raise InvalidParams("Only one 'orderBy' is supported")
 
-        orderby = self.orderby[0]
-        self._validate_field(orderby.field)
+        for orderby in self.orderby:
+            self._validate_field(orderby.field)
+
+        orderby_fields, orderby_operations = set(), set()
+        for f in self.orderby:
+            orderby_fields.add(f.field)
+            if f.field.op:
+                orderby_operations.add(f.field.op)
+        self._validate_snuba_functions_in_one_group(orderby_operations)
+
+        # validate all orderby columns are presented in provided 'fields'
+        if set(self.select).issuperset(orderby_fields):
+            return
 
-        for select_field in self.select:
-            if select_field == orderby.field:
-                return
         raise InvalidParams("'orderBy' must be one of the provided 'fields'")
 
+    @staticmethod
+    def _validate_snuba_functions_in_one_group(orderby_operations: Set[str]) -> None:
+        for snuba_func in OP_TO_SNUBA_FUNCTION.keys():
+            valid_operations = set(OP_TO_SNUBA_FUNCTION[snuba_func].keys())
+            if orderby_operations.issubset(valid_operations):
+                return
```

Comment:
You can construct a metrics expression through calling `metric_object_factory` https://github.com/getsentry/sentry/blob/b26243f8d9858b47e13248984194809164ecb379/src/sentry/snuba/metrics/fields/base.py#L1127 and then you can call `get_entity` to get the entities of the metric expressions, and if that set contains more than one entity then raise an Exception